frontend/app/src/router/index.ts

```diff
@@ -50,7 +50,7 @@ router.beforeEach((to, from, next) => {
 
     next();
   }
-  else if (to.path.startsWith('/user')) {
+  else if (to.path.startsWith('/user') || to.path === '/wallet-bridge') {
```

Comment:
this one to make if the `/wallet-bridge` route opened in the browser, it bypass the auth